invenio_accounts/cli.py

```diff
@@ -57,6 +59,8 @@ def users_create(email, password, active):
     if form.validate():
         kwargs["password"] = hash_password(kwargs["password"])
         kwargs["active"] = active
+        if confirm:
+            kwargs["confirmed_at"] = date.today()
```

Comment:
To be consistent with other timestampts stored in the DB (e.g. created/updated) it should contain the h/m/s too.
```suggestion
            kwargs["confirmed_at"] = datetime.utcnow()
```

Reply:
should we use `datetime.utcnow.isoformat()` as in https://github.com/inveniosoftware/invenio-saml/blob/1aca07d56ebec7dedd02b15ba7c7192840751493/invenio_saml/handlers.py#L69 ?

further: https://github.com/inveniosoftware/invenio-saml/pull/31#discussion_r1003742387

Reply:
I would personally keep it as a datetime and then let the DB handle representations (imagine someone does not want ISO format for some reason). That would also make it consistent with the ORM. In [records](https://github.com/inveniosoftware/invenio-records/blob/master/invenio_records/models.py#L31) we are using `utcnow` as default.

Reply:
ok, then i go with `datetime.utcnow()` 
i will reference this decision also in the invenio-saml PR. so we could do the same there too